packages/gcp/src/__tests__/ServiceWrapper.test.ts

```diff
@@ -3,33 +3,36 @@
  */
 
 import { ProjectsClient } from '@google-cloud/resource-manager'
-import { compute_v1 } from 'googleapis'
-import { compute as googleCompute } from 'googleapis/build/src/apis/compute'
+import {
+  AddressesClient,
+  DisksClient,
+  ImagesClient,
+  InstancesClient,
+  MachineTypesClient,
+  protos as googleCompute,
+} from '@google-cloud/compute'
 import { GoogleAuth } from 'google-auth-library'
 import { RecommenderClient } from '@google-cloud/recommender'
 import { GoogleAuthClient, wait } from '@cloud-carbon-footprint/common'
 import {
-  InstanceData,
-  mockedAddressesResultItems,
+  mockAddressesResultItems,
   mockedAddressGetDetails,
   mockedDisksGetSSDDetails,
-  mockedDisksResultItems,
+  mockDisksResultItems,
   mockedImageGetDetails,
   mockedInstanceGetItems,
-  mockedInstanceResultItems,
+  mockInstanceResultItems,
   mockedMachineTypesGetItems,
 } from './fixtures/googleapis.fixtures'
-import Schema$Instance = compute_v1.Schema$Instance
-import Schema$MachineType = compute_v1.Schema$MachineType
-
+import { mockedProjects } from './fixtures/resourceManager.fixtures'
+import { mockStopVMRecommendationsResults } from './fixtures/recommender.fixtures'
+import Instance = googleCompute.google.cloud.compute.v1.IInstance
+import MachineType = googleCompute.google.cloud.compute.v1.IMachineType
```

Comment:
Is this the correct way to import? or could it be directly from `@google-cloud/compute`? If so, seems fine.

Reply:
I haven't had much luck importing it directly. It follows the same structure as the googleapis, where you import the prototypes from the package:
`import { protos as googleCompute } from '@google-cloud/compute'`
then import the type/interface from that object.